xDrip/Resources/en.lproj/Localizable.strings

```diff
@@ -19,4 +19,24 @@
 "entries_list_data_section_title" = "DATA";
 "entries_list_scene_title_carbs" = "Active Carbohydrates";
 "entries_list_scene_title_bolus" = "Active insulin";
-"entries_list_cell_no_date" = "No date specified";
\ No newline at end of file
+"entries_list_cell_no_date" = "No date specified";
+
+// Settings
+"settings_title" = "Settings";
```

Comment:
There would be a lot of settings strings, we need to group them in a better way. Rename this section to `Settings Root` and use `settings_root_title_` as the prefix.